data/human/culture republic.txt

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+#mission "ZZCU cultural holiday template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		and
+#			month == placeholder
+#			day >= placeholder
+#			day <= placeholder
+#	on offer
+#		conversation
+#			`placeholder`
+#				decline
+			
+#mission "ZZCU cultural random template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		random == 1
+#	on offer
+#		conversation
+#			`placeholder`
+#				decline
+
+
+#############################################
+#      Republic with Syndicate Pairing      #
+#############################################
+
+mission "ZZCU Sad Gamblers -R"
```

Comment:
You basically have this same thing in two different regions. Maybe have some difference between the regions? I'd also suggest changing the names. Actually type out "Republic" and "Syndicate" instead of "-R" and "-S".

Reply:
I was trying to get it to appear anywhere in either region but I couldn't get the source criteria to work right.